### PR TITLE
More visible warning/error

### PR DESCRIPTION
**Pull request by BrennanConroy**
#1051 
color

color2

### CODE REVIEW

**Comment by muratg** [comment]
:shipit:

File: src/Microsoft.Framework.PackageManager/Restore/NuGet/NuGetv2Feed.cs [hunk 1]
```diff
@@ -151,18 +151,18 @@ public Task<IEnumerable<PackageInfo>> FindPackagesByIdAsync(string id)
                         {
                             _ignored = true;
                             _reports.Information.WriteLine(
-                                string.Format("Failed to retrieve information from remote source '{0}'".Yellow(),
+                                string.Format("Failed to retrieve information from remote source '{0}'".Yellow().Bold(),
```
Comment (BrennanConroy) [inline src/Microsoft.Framework.PackageManager/Restore/NuGet/NuGetv2Feed.cs]: move this to append string.Format like the others

**Comment by BrennanConroy** [comment]
@Praburaj

**Comment by davidfowl** [comment]
:shipit:

**Comment by davidfowl** [comment]
Thought you might want to test it in the VS output to see how it looks

**Comment by Praburaj** [comment]
Looks good to me :shipit:

**Comment by BrennanConroy** [comment]
VS output doesn't do color (if it does please tell me how!) so it is unaffected by this change.

**Pull request closed without merging**